app/src/pages/projects/ProjectsPage.tsx

```diff
@@ -32,9 +55,20 @@ export function ProjectsPage() {
         }
       }
     `,
-    {}
+    data
   );
-  const projects = data.projects.edges.map((p) => p.project);
+  const projects = projectsData.projects.edges.map((p) => p.project);
+
+  // Refetch projects if the fetchKey changes
+  useEffect(() => {
+    startTransition(() => {
+      refetch({}, { fetchPolicy: "store-and-network" });
+    });
+  }, [fetchKey, refetch]);
+
+  const canDeleteProjects = useMemo(() => {
+    return projects.length > 1;
+  }, [projects]);
```

Comment:
I think this is kinda arbitrary - we should probably start with just the default being non-deletable

Reply:
addressed in a78866b1